android/BOINC/app/src/main/AndroidManifest.xml

```diff
@@ -29,10 +29,6 @@
         android:smallScreens="true"
         android:xlargeScreens="true" />
 
-    <uses-sdk
-        android:minSdkVersion="19"
-        android:targetSdkVersion="26" />
-
```

Comment:
@adamradocz 
I'm not aware about general Android application build configuration but how we can show that we support not the latest Android OS but also some previous one?
How Play Store, user and actually we can know what is the minimal required SDK version that is supported by BOINC Manager?